### PR TITLE
lightning: update config description of checksum and analyze

### PR DESCRIPTION
**Pull request by glorv**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Update outdated config description for lightning.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-lightning/tidb-lightning-configuration.md [hunk 1]
```diff
@@ -111,12 +111,12 @@ addr = "172.16.31.10:8287"
 
 [mydumper]
 # 设置文件读取的区块大小，确保该值比数据源的最长字符串长。
-read-block-size = 65536 # Byte (默认为 64 KB)
+read-block-size = "64KiB" (默认为 64 KiB)
```
Comment (kennytm) [inline tidb-lightning/tidb-lightning-configuration.md:114]:
```suggestion
read-block-size = "64KiB"  # 默认值
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 2]
```diff
@@ -160,7 +160,7 @@ data-invalid-char-replace = "\uFFFD"
 strict-format = false
 
 # 如果 strict-format = true，TiDB Lightning 会将 CSV 大文件分割为多个文件块进行并行处理。max-region-size 是分割后每个文件块的最大大小。
-# max-region-size = 268_435_456 # Byte（默认是 256 MB）
+# max-region-size = "256MiB" （默认是 256 MiB）
```
Comment (kennytm) [inline tidb-lightning/tidb-lightning-configuration.md:163]: ditto

**Review by kennytm — COMMENTED** [review]
rest lgtm

File: tidb-lightning/tidb-lightning-configuration.md [hunk 3]
```diff
@@ -234,17 +234,22 @@ max-allowed-packet = 67_108_864
...
 # 执行的顺序为：Checksum -> Compact -> Analyze。
 [post-restore]
-# 如果设置为 true，会对所有表逐个执行 `ADMIN CHECKSUM TABLE <table>` 操作
-# 来验证数据的完整性。
-checksum = true
+# 配置是否在导入完成后对每一个表执行 `ADMIN CHECKSUM TABLE <table>` 操作来验证数据的完整性。
+# 可选的配置项：
+# - "required"（默认）。在导入完成后执行 CHECKSUM 检查，如果 CHECKSUM 检查失败，则会报错退出
+# - "optional"。在导入完成后执行 CHECKSUM 检查，如果报错，会输出一条 WARN 日志并忽略错误
+# - "off"。导入结束后不执行 CHECKSUM 检查。
```
Comment (Liuxiaozhen12) [inline tidb-lightning/tidb-lightning-configuration.md:241]:
```suggestion
# - "required"（默认）：在导入完成后执行 CHECKSUM 检查，如果 CHECKSUM 检查失败，则会报错退出。
# - "optional"：在导入完成后执行 CHECKSUM 检查，如果报错，会输出一条 WARN 日志并忽略错误。
# - "off"：导入结束后不执行 CHECKSUM 检查。
```

**Review by Liuxiaozhen12 — APPROVED** [review]

File: tidb-lightning/tidb-lightning-configuration.md [hunk 4]
```diff
@@ -373,8 +368,13 @@ min-available-ratio = 0.05
 | --tidb-password *password* | 连接到 TiDB 的密码 | `tidb.password` |
 | --no-schema | 忽略表结构文件，直接从 TiDB 中获取表结构信息 | `mydumper.no-schema` |
 | --enable-checkpoint *bool* | 是否启用断点 (默认值为 true) | `checkpoint.enable` |
+<<<<<<< HEAD
+| --analyze *level* | 导入后分析表信息，可选值为 required、optional、off (默认为 required) | `post-restore.analyze` |
+| --checksum *level* | 导入后比较校验和，可选值为 required、optional、off (默认为 optional) | `post-restore.checksum` |
+=======
```
Comment (TomShawn) [inline tidb-lightning/tidb-lightning-configuration.md:374]: @glorv Please resolve the conflict.

**Review by TomShawn — APPROVED** [review]

**Comment by TomShawn** [comment]
@hfxsd PTAL and have it merged after you approve it. Thanks~

**Review by hfxsd — APPROVED** [review]
LGTM

**Pull request merged**